eng/WpfArcadeSdk/tools/ShippingProjects.props

```diff
@@ -5,7 +5,6 @@
       PresentationBuildTasks;
       PresentationCore;
       DirectWriteForwarder;
-      PresentationCore-CommonResources;
```

Comment:
I don't see a removal from `<UseMicrosoftSharedKeyId>` section. Looks like we have always been missing `PresentationCore-CommonResources` from that section. 

Becasue`PresentationCore-CommonResources`  was a _breakaway_ from `PresentationCore`, I we should have reasoned our way into assigning it the same _key_ as `PresentationCore`...

Reply:
Yeah, I just removed what reference we had.  Like we talked about, I think it's fine to leave the repo alone until next week when we can check this in without messing with Preview6 and the concern will go away.